src/Npgsql/NpgsqlSchema.cs

```diff
@@ -438,6 +439,291 @@ public static DataTable GetReservedWords()
+            {"TIMESTAMP", (int)NpgsqlDbType.Timestamp, 8, "TIMESTAMP({0})",
+                    "precision of fractional seconds", "System.DateTime",
+                    false, false, false, true, true,
+                    false, true, true, false, DBNull.Value,
+                    DBNull.Value, DBNull.Value,
+                    false, true, "TIMESTAMP '", "'", "DBTYPE_DBTIMESTAMP"},
+            {"INTERVAL", (int)NpgsqlDbType.Interval, 16, "INTERVAL {0}({1})",
+                    "quantity, unit", "System.TimeSpan",
+                    false, false, false, true, true,
+                    false, true, true, false, DBNull.Value,
+                    DBNull.Value, DBNull.Value,
+                    false, true,"INTERVAL '", "'", "DBTYPE_DBTIME"},
+            {"BYTEA",(int)NpgsqlDbType.Bytea, DBNull.Value, "BYTEA",
+                    DBNull.Value, "System.Byte[]",
+                    false, false, false, false, false,
+                    false, true, false, false,DBNull.Value,
+                    DBNull.Value, DBNull.Value, false, true,
+                    "BYTEA '", "'","DBTYPE_BYTES"},
+            {"TEXT", (int)NpgsqlDbType.Text, DBNull.Value, "TEXT",
+                    DBNull.Value, "System.String",
+                    false, true, true, false, false,
+                    false, true, true, true, DBNull.Value,
+                    DBNull.Value, DBNull.Value,
+                    false, true, "'", "'", "DBTYPE_STR"},
+            {"CITEXT", (int)NpgsqlDbType.Citext, DBNull.Value, "CITEXT",
+                    DBNull.Value, "System.String",
+                    false, false, true, false, false,
+                    false, true, true, true, DBNull.Value,
+                    DBNull.Value, DBNull.Value,
+                    false, true, "'", "'", "DBTYPE_STR"},
+            {"BOOLEAN", (int)NpgsqlDbType.Boolean, 1, "BOOLEAN",
+                    DBNull.Value,"System.Boolean",
+                    false, true, false, true, false,
+                    false, true, true, false, DBNull.Value,
+                    DBNull.Value, DBNull.Value, false, true,
+                    DBNull.Value, DBNull.Value, "DBTYPE_BOOL"},
+            {"BIT", (int)NpgsqlDbType.Bit, DBNull.Value, "BIT({0})",
+                    "length", "System.Collections.BitArray",
+                    false, true, false, true, true,
+                    false, true, true, false, DBNull.Value,
+                    DBNull.Value, DBNull.Value, false, true,
+                    "B'", "'", "DBTYPE_BYTES"},
+            {"MONEY", (int)NpgsqlDbType.Money, 8, "MONEY",
+                    DBNull.Value,"System.Decimal",
+                    false, false, false, true, false,
+                    false, true, true, false, DBNull.Value,
+                    DBNull.Value, DBNull.Value,false, true,
+                    "MONEY '", "'", "DBTYPE_CY"},
+            {"BOX", (int)NpgsqlDbType.Box, 32, "BOX",
+                    DBNull.Value, "NpgsqlBox",
```

Comment:
Shouldn't the .NET type name by fully-qualified (same with all others below)?

Reply:
By the way, use `nameof(T)` if you want to get the type name without the namespace or `typeof(T).FullName` to get the full name. These expressions protect you from making typos.